shell/list/workload.vue

```diff
@@ -151,5 +185,14 @@ export default {
 </script>
 
 <template>
-  <ResourceTable :loading="$fetchState.pending" :schema="schema" :rows="filteredRows" :overflow-y="true" />
+  <div>
+    <ResourceTable
+      :loading="$fetchState.pending"
+      :schema="schema"
+      :rows="filteredRows"
+      :overflow-y="true"
+      :set-page-fn="setPage"
```

Comment:
ResourceTable uses resourcelist, it is using all of that plumbing that I built in elsewhere. I don't need the outer div though, that was for adding some debugging that I don't need anymore and I can get rid of it.

Reply:
The simple answer is that it does :) I must've missed that because sorting and searching is apparently broken on workloads as a result of its absence.